modules/ROOT/pages/rtfctl-permissions.adoc

```diff
@@ -0,0 +1,112 @@
+= RTFCTL Kubernetes Permissions
+
+Understanding and enabling the specific permissions required over the Kubernetes resources is crucial for successfully running `rtfctl` commands. Properly configured role permissions are essential for seamless execution of `rtfctl` operations.
+
+The following table lists all the permissions that you configure using K8s RBACs (https://kubernetes.io/docs/reference/access-authn-authz/rbac/[Role based access control^]) in Kubernetes: 
+
+* Namespace +
+The Namespace column indicates that the permission can be a Role based or ClusterRole based:
+ ** Role based: the namespace can be for example, `rtf`, `<app-namespace>`, `rtf-validate`, or `kube-node-lease`.
+ ** ClusterRole: the namespace field doesn't exist there because it applies to the entire cluster, thus blank in the table.
+
+* apiGroups +
+The apiGroups column indicates the apiGroup that group K8s resources. When you create a Role, each K8s resource declares what is their apiGroups. When blank in the table, defaults to the core apiGroup. Refer to https://kubernetes.io/docs/reference/using-api/#api-groups[API groups^] for details.
```

Comment:
Read this out loud: The apiGroups column indicates the apiGroup that group K8s resources. 

Maybe:
The apiGroups column indicates the apiGroup for the K8s resources. 


Another verb for group here? If group is correct, change to groups. (apiGroup groups)